src/NHibernate/NHibernate.csproj

```diff
@@ -73,6 +73,10 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))">
+    <PackageReference Include="Backport.System.Threading.Lock" Version="1.1.6" />
```

Comment:
So, that is [this Nuget package](https://www.nuget.org/packages/Backport.System.Threading.Lock), coming from [your repository](https://github.com/MarkCiliaVincenti/Backport.System.Threading.Lock).

Reply:
Yes, that is correct.